app/views/pages/home.html.erb

```diff
@@ -1,27 +1,35 @@
-<section>
-  <nav>
-  <% if user_signed_in? %>
```

Comment:
Are you going to add an option to signout/access the dashboard based upon whether the user is signed in or not? If so, it is OK to delete this :)